index.bs

```diff
@@ -7471,12 +7474,11 @@ value when its bit pattern is interpreted as an unsigned 64 bit integer.
 
 <div id="USVString-to-es" algorithm="convert an USVString to an ECMAScript value">
 
-    An IDL {{USVString}} value is [=converted to an ECMAScript value|converted=]
-    to an ECMAScript value by running the following algorithm:
-
-    1.  Let |scalarValues| be the sequence of [=scalar values=] the {{USVString}} represents.
-    1.  Let |string| be the sequence of [=code units=] that results from encoding |scalarValues| in UTF-16.
-    1.  Return the String value that represents the same sequence of [=code units=] as |string|.
+    The result of [=converted to an ECMAScript value|converting=]
+    an IDL {{USVString}} value to an ECMAScript
+    value is the String
+    value that represents the sequence of [=code units=]
+    in the IDL {{USVString}}.
```

Comment:
Do we have to consider the String length limit in ECMAScript at all? https://tc39.es/ecma262/#sec-ecmascript-language-types-string-type

Reply:
Do we have to define this at all given how Infra defines them to be compatible?

Reply:
We have to define something, but I can simplify this further.